core/examples/luigi-sample-angular/src/luigi-config/extended/settings.js

```diff
@@ -14,7 +14,6 @@ class Settings {
   responsiveNavigation = 'simpleMobileOnly'; // Options: simple | simpleMobileOnly | semiCollapsible
   sideNavFooterText = `Luigi Client: ${version || 'unknown'}`;
   thirdPartyCookieCheck = {
-    thirdPartyCookieDetection: true,
```

Comment:
Since you commented out the `thirdPartyCookieScriptLocation` it might make sense to also just comment out that line?